realsense2_camera/CMakeLists.txt

```diff
@@ -106,32 +105,23 @@ if(NOT realsense2_FOUND)
 endif()
 
 #set(CMAKE_NO_SYSTEM_FROM_IMPORTED true)
-include_directories(
-    include
-    ${realsense2_INCLUDE_DIR}
-    )
+include_directories(include)
 
 set(node_plugins "")
 
 add_library(${PROJECT_NAME} SHARED
-    include/constants.h
-    include/realsense_node_factory.h
-    include/base_realsense_node.h
-    include/t265_realsense_node.h
     src/realsense_node_factory.cpp
     src/base_realsense_node.cpp
     src/t265_realsense_node.cpp
-    )
-
+)
 
 target_link_libraries(${PROJECT_NAME}
     ${realsense2_LIBRARY}
     )
```

Comment:
This shouldn't be nessesary. It should be handled by `ament_target_dependencies`. However, after the rebase i couldn't get it to work, so i had to leave it.